client/my-sites/menus/controller.js

```diff
@@ -3,7 +3,8 @@
  */
 var ReactDom = require( 'react-dom' ),
 	React = require( 'react'),
-	i18n = require( 'i18n-calypso' );
+	i18n = require( 'i18n-calypso' ),
+	ReactRedux = require( 'react-redux' );
```

Comment:
I think we've been generally okay with mixing import styles, avoiding adding new `require` at all. It's ugly, but I guess you can argue that the ugliness will hasten the transition 😉